lib/codecs/src/encoding/format/syslog.rs

```diff
@@ -0,0 +1,577 @@
+        // Does not seem to retrieve `%vector.ingest_timestamp`?
+        // - A sink type `console` with `timestamp_format: unix_ms` converts a `Value::Timestamp` prior to the encoder logic
+        // to `Value::Integer(i64)` instead, which won't match this condition.
+        //
+        // NOTE:
+        // Vector always manages `Value::Timestamp` as `DateTime<Utc>`, any prior TZ information context is always dropped.
+        // If restoring the TZ for a log is important, it could be handled via a remap transform?
+        //
+        // Ref:
+        // `log.get_timestamp()`:
+        // https://github.com/vectordotdev/vector/blob/ad6a48efc0f79b2c18a5c1394e5d8603fdfd1bab/lib/vector-core/src/event/log_event.rs#L543-L552
+        if let Some(Value::Timestamp(timestamp)) = self.log.get_timestamp() {
+            *timestamp
+        } else {
+            // NOTE:
+            // When timezone information is missing Vector handles conversion to UTC by assuming the local TZ:
+            // https://vector.dev/docs/about/under-the-hood/architecture/data-model/log/#time-zones
+            // There is a global option for which TZ to assume (where the default is local TZ):
+            // https://vector.dev/docs/reference/configuration/global-options/#timezone
+            // https://github.com/vectordotdev/vector/blob/58a4a2ef52e606c0f9b9fa975cf114b661300584/lib/vector-core/src/config/global_options.rs#L233-L236
+            // https://github.com/vectordotdev/vrl/blob/c010300710a00191cd406e57cd0f3e001923d598/src/compiler/datetime.rs#L88-L95
+            // VRL remap can also override that:
+            // https://vector.dev/docs/reference/configuration/transforms/remap/#timezone
+            // Vector's `syslog` source type also uses `Utc::now()` internally as a fallback:
+            // https://github.com/vectordotdev/vector/blob/58a4a2ef52e606c0f9b9fa975cf114b661300584/src/sources/syslog.rs#L430-L438
+            Utc::now()
+        }
+    }
+
+    fn get_message(&self, config: &SyslogSerializerOptions) -> String {
+        // `payload_key` configures where to source the value for the syslog `message`:
+        // - Not configured      => Encodes the default log message.
+        // - Field key (Valid)   => Get value by lookup (value_by_key)
+        // - Field key (Invalid) => Empty string (unwrap_or_default)
+
+        // Ref:
+        // `log.get_message()`:
+        // https://github.com/vectordotdev/vector/blob/ad6a48efc0f79b2c18a5c1394e5d8603fdfd1bab/lib/vector-core/src/event/log_event.rs#L532-L541
+        // `v.to_string_lossy()`:
+        // https://github.com/vectordotdev/vrl/blob/f2d71cd26cb8270230f531945d7dee4929235905/src/value/value/serde.rs#L34-L55
+        let payload = if config.payload_key.is_empty() {
+            self.log.get_message().map(|v| v.to_string_lossy().to_string() )
+        } else {
+            self.value_by_key(&config.payload_key)
+        };
+
+        payload.unwrap_or_default()
```

Comment:
No payload key, so just something like this should do:

```suggestion
            self.log.get_message().map(|v| v.to_string_lossy().to_string())
```